src/routing_node.rs

```diff
@@ -402,6 +407,19 @@ impl RoutingNode {
             return Err(RoutingError::FilterCheckFailed);
         }
 
+        let message = signed_message.get_routing_message().clone();
+
+        // Cache a response if from a GetRequest and caching is enabled for the Data type.
+        self.handle_cache_put(&message);
+        // Get from cache if it's there.
+        match self.handle_cache_get(&message) {
+            Some(content) => {
+                return self.send_content(
```

Comment:
current practice is not good either, but it would currently be written
let _ = self.send_content(...);
return Ok(())

but this is just a note, and not needed to be changed.